src/App/Components/Profile/BookMark.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import _ from 'lodash';
```

Comment:
@alainburindi `lodash` is a very huge module. please, import only what you need from it.
Like this `import isEmpty from 'lodash/isEmpty';`